src/pages/UserEnrollments/Enrollment.css

```diff
@@ -0,0 +1,77 @@
+.enrollment {
+  width: 100%;
+  box-shadow: 0 1px 4px 0 #C9D3DF;
+  background-color: var(--white);
+}
+
+.enrollment-header {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: wrap;
+  align-items: center;
+  padding: 1em;
+  border-bottom: 1px solid var(--light-grey);
+}
+
+.enrollment-header .fs {
+  display: flex;
+  align-items: center;
+  font-size: 1.25em;
+  font-weight: 600;
+}
+
+.enrollment-header .fs img {
+  height: 40px;
+  margin: 0 0.5em;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-flow: wrap;
+}
+
+.intitule {
+  font-weight: 600;
+  font-size: large;
+}
+
+.enrollment-body {
+  padding: 1em;
+}
+
+.enrollment-footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 2em;
+}
+
+.tag {
+  padding: 0.5em 1em;
+  text-align: center;
+  border-radius: 8px;
```

Comment:
This introduce a new button style. We have 2 options:

1. upgrade button style on the whole application
2. use the old style until we do the first point

We should not have the 2 style cohabiting at the same time

Reply:
It's not a button, so it's normal that it doesn't look like one.